helm-chart/build.py

```diff
@@ -115,7 +121,38 @@ def publish_pages():
     subprocess.check_call(
         ['git', 'push', 'origin', 'gh-pages'],
         cwd='gh-pages',
-        env=dict(os.environ, GIT_SSH_COMMAND='ssh -i ../travis')
+        env=dict(os.environ, GIT_SSH_COMMAND=f'ssh -i ../{HELM_CHART_DEPLOY_KEY_NAME}')
+    )
+
+def update_mybinder_deployment(version):
+    commit = last_git_modified('.')
+    subprocess.check_call([
+        'git', 'clone',
+        'https://github.com/jupyterhub/mybinder.org-deploy', 'staging'],
+        env=dict(os.environ, GIT_SSH_COMMAND='ssh -i {MYBINDER_DEPLOY_KEY_NAME}')
+    )
+    subprocess.check_call(['git', 'checkout', 'staging'], cwd='staging')
+    subprocess.check_call(['git', 'remote', 'add', 'meeseeks', 'ssh://git@github.com/MeeseeksBox/mybinder.org-deploy'])
+
+    with open(os.path.join('staging', 'config', 'common.yaml')) as f:
```

Comment:
when I moved projects yesterday, the location for the binderhub version has also changed. I haven't updated README yet, but the version is now in mybinder/requirements.yaml. Apologies for changing this out from under you!